pkgs/development/python-modules/authres/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, openssl, makeWrapper, buildPythonApplication
+, pytest, dnspython, pynacl, authres }:
+
+buildPythonApplication rec {
+  pname = "authres";
+  majorversion = "1.0";
+  minorversion = "1";
```

Comment:
these two are not parameters of `buildPythonApplication` and thus best moved into a `let ... in` expression.